### PR TITLE
Added support for bazel files as plain test

### PR DESCRIPTION
**Pull request by acecilia**
👋 

As the title says.

Note that during development I found multiple issues when building from code that I reported in https://github.com/sbarex/SourceCodeSyntaxHighlight/issues/230

### CODE REVIEW

**Comment by acecilia** [comment]
Isn't there any way to maybe autogenerate Info.plist files? At the moment adding support for a new file type is quite convoluted. See this PR for example: adding plain-text support to couple of new files requires several added lines, and those lines without this PR are difficult to understand

File: Application/resources/settings.yaml [hunk 1]
```diff
@@ -368,6 +379,12 @@ extensions:
         
     do:
         syntax: txt
+
+    bazel:
+        syntax: txt
+
+    bzl:
+        syntax: txt
```
Comment (acecilia) [inline Application/resources/settings.yaml:387]: Is this needed?

File: Application/Info.plist [hunk 2]
```diff
@@ -2709,6 +2709,26 @@
...
+			<key>UTTypeConformsTo</key>
+			<array>
+				<string>public.item</string>
+				<string>dyn.ah62d4rv4ge80e2p4qz0a</string>
+				<string>dyn.ah62d4rv4ge80e8xq</string>
+			</array>
+			<key>UTTypeDescription</key>
+			<string>Bazel file</string>
+			<key>UTTypeIdentifier</key>
+			<string>org.bazel.source</string>
```
Comment (acecilia) [inline Application/Info.plist:2722]: I chose this myself arbitrarily, is this okey?

File: QLExtension/Info.plist [hunk 3]
```diff
@@ -116,6 +116,7 @@
 				<string>org.arduino.source</string>
 				<string>org.asm.source</string>
 				<string>org.bash.source</string>
+				<string>org.bazel.source</string>
```
Comment (acecilia) [inline QLExtension/Info.plist:119]: Is this needed? Why?

Comment (sbarex) [reply QLExtension/Info.plist:119]: The `QLExtension/Info.plist` is mandatory for the quicklook extension to manage the file.
The `Application/Info.plist` define the custom UTI properties (like the file extensions).

**Comment by acecilia** [comment]
@sbarex Is this ready to merge?

**Comment by acecilia** [comment]
@sbarex Why was this closed 🤔

**Pull request closed without merging**